extensions/wikia/AffiliateService/styles/affiliate-unit.scss

```diff
@@ -85,6 +85,16 @@
         position: relative;
     }
 
+    .WikiaArticle p &__extra-disclaimer-message {
```

Comment:
Looks like the same styles from `&__disclaimer-message`.  Do we need to keep these styles separate?

Reply:
Potentially, I figured the flexibility to do so outweighed the assumption that we wouldn't want to...

Reply:
and then I just noticed that I didn't even end up using in to the template... D'oh!. Removing...